res/database/Default/user/scr0llbaer.xml

```diff
@@ -0,0 +1,1586 @@
+					<title>
+						<de>Aliens haben mein Kindermädchen gefressen</de>
+						<en>Aliens ate my nanny</en>
+					</title>
+					<description>
+						<de>Wieder steckte Bobbys böswilliger Erzfeind aus der Schule, der einen IQ-Punkt mehr hat, hinter allem und kontrollierte all die Alien-Rassen. Und dieser macht auch einfach nicht halt und will diesmal gleich das ganze Universum zerstören. Der große Showdown beginnt...</de>
+						<en>Again, Bobby's malicious nemesis from school, who has one IQ point more, was behind everything and controlled all the alien races. And he just doesn't stop and wants to destroy the whole universe this time. The big showdown begins...</en>
+					</description>
+				</scripttemplate>
+			</children>
+
+			<jobs>
+				<job index="0" function="1" required="1" />
+				<job index="1" function="2" required="1" gender="1" role_guid="437539e5-4c6b-4207-981a-051a82021745" />
+				<job index="2" function="2" required="1" gender="0" role_guid="ba05561a-0266-4cfe-b020-353781fd881c" />
+				<job index="3" function="2" required="1" gender="1" />
+				<job index="4" function="2" required="1" gender="2" />
+				<job index="5" function="32" required="1" gender="1" />
+				<job index="6" function="32" required="1" gender="2" />
+			</jobs>
+
+			<genres maingenre="3" subgenres="16,1" />
+			<episodes min="8" max="8" />
+			<blocks value="1" />
+			<price min="9000" max="14000" />
+			<potential min="50" max="70" />
+			<outcome min="50" max="70" />
+			<review min="60" max="70" />
+			<speed min="50" max="70" />
+		</scripttemplate>
+
+		<scripttemplate product="2" licence_type="3" guid="c0d67fc9-1388-4715-b708-713df17c681a" creator="9551" comment="inspired from https://en.wikipedia.org/wiki/Leisure_Suit_Larry">
+			<title>
+				<de>Leather Jacket Leo, %ladykiller%</de>
+				<en>Leather Jacket Leo, %ladykiller%</en>
+			</title>
+			<description>
+				<de>Leo ist bis Mitte 30 unberührt und unschuldig. Nun will er's aber wissen und sucht eine Absteige auf. Nach einigen Schwierigkeiten... Endlich! Langsam kriegt er den Dreh raus, und beginnt dann nach der Frau seines Lebens zu suchen.</de>
+				<en>Leo is untouched and innocent until his mid-30s. But now he wants to get it done and seeks out a flophouse. After some difficulties... Finally! Slowly he gets the hang of it, and then starts looking for the woman of his life.</en>
+			</description>
+
+			<variables>
+				<ladykiller>
+					<de>Ladykiller|Schwerenöter vom Dienst|Reizender Charmeur|Bezaubernder Schmeichler</de>
+					<en>Ladykiller|Philanderer at your Service|Endearing Charmer|Adorable Sweet-Talker</en>
+				</ladykiller>
+			</variables>
+
+			<children>
+				<scripttemplate index="0" product="2" licence_type="2" guid="8f5a2c84-89b4-41fe-9e27-3898382f612b">
+					<title>
+						<de>Leather Jacket Leo from the Land of the Lone Lobsters</de>
+						<en>Leather Jacket Leo from the Land of the Lone Lobsters</en>
```

Comment:
land lower case

Reply:
Nope, it is a custom name - like "Manhattan Island" or names like "Land of the Free"

Reply:
more because of the title casing.

spoof of https://en.wikipedia.org/wiki/Leisure_Suit_Larry_in_the_Land_of_the_Lounge_Lizards